oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandlerTest.java

```diff
@@ -94,6 +94,10 @@ class AuthenticationCallbackHandlerTest {
                     .setVerifiedMfaMethodType(MFAMethodType.EMAIL)
                     .setAuthenticated(false)
                     .setCurrentCredentialStrength(null);
+    private static final OrchSessionItem ORCH_SESSION =
```

Comment:
Neither the session nor the client session use screaming snake case. Either update them to use it, or do camelCase for orch session. I think the latter, as that's what other tests do.